flask-redis-centos7-atomicapp/artifacts/kubernetes/02-service-flask.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: flask
+  labels:
+    name: flask
+spec:
+  ports: 
+  - port: 5000
```

Comment:
you can use something like `nodePort` field here and specify the port on the node (yes using env vars) where the service will be accessible.

Check https://github.com/projectatomic/nulecule-library/blob/master/gitlab-centos7-atomicapp/artifacts/kubernetes/gitlab-http-service.json